include/ipmitool/Makefile.am

```diff
@@ -39,4 +39,4 @@ noinst_HEADERS = log.h bswap.h hpm2.h helper.h ipmi.h ipmi_cc.h ipmi_intf.h \
 	ipmi_fwum.h ipmi_main.h ipmi_tsol.h ipmi_firewall.h \
 	ipmi_kontronoem.h ipmi_ekanalyzer.h ipmi_gendev.h ipmi_ime.h \
 	ipmi_delloem.h ipmi_dcmi.h ipmi_vita.h ipmi_sel_supermicro.h \
-	ipmi_cfgp.h ipmi_lanp6.h ipmi_quantaoem.h
+	ipmi_cfgp.h ipmi_lanp6.h ipmi_quantaoem.h ipmi_time.h
```

Comment:
Looks like this change need to be in a separate PR as it is not directly related to RPM.

Reply:
Yes, not directly related to RPM, but I saw the problem when attempting to build an RPM. Can this problem be seen in another case ?

Reply:
I've put this in a separate commit